### PR TITLE
Cursor autofocus set after changing the currency on request/split money screen

### PR DESCRIPTION
**Pull request by PrashantMangukiya**
@michaelhaxhiu pr is ready for review.

### Details
On Request / Split money screen cursor autofocus was not regained after changing the currency. Now updated code to set cursor autofocus after changing the currency.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5102

### Tests and QA Steps
1. Launch the desktop app and login
2. Click the + icon in LHN and choose either request or split money
3. Click the currency and choose any other currency
4. After selecting new currency now it will regained focus to amount field.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

https://user-images.githubusercontent.com/7823358/133469891-6602c4a4-d45b-433d-96df-83475d50b426.mov

#### Mobile Web

https://user-images.githubusercontent.com/7823358/133470142-107aaecb-0e41-474d-99fc-499054cc9dcb.mov

#### Desktop

https://user-images.githubusercontent.com/7823358/133470611-c6eacd9b-14c8-4c26-9297-61024177e2fe.mov

#### iOS

https://user-images.githubusercontent.com/7823358/133470765-3f4b1298-de6c-4313-b6da-dfc657bd0f3a.mov

#### Android

https://user-images.githubusercontent.com/7823358/133471079-cf2fe564-4f1d-48b7-8e4d-6ee9ed694161.mov

### CODE REVIEW

**Comment by michaelhaxhiu** [comment]
Thanks for the quick work @PrashantMangukiya. Next step is to await @pecanoro's review of the PR, and address any comments for adjustments/changes.

**Comment by kadiealexander** [comment]
Hi @PrashantMangukiya, I've placed this issue on hold as per [this update](https://expensify.slack.com/archives/C01GTK53T8Q/p1631754482374800), as we are prioritising issues related to a feature release scheduled for 9/24. As an apology for the delay, we will add a $100 bonus as a thank you for waiting.

**Review by pecanoro — APPROVED** [review]

**Comment by PrashantMangukiya** [comment]
@pecanoro After merging main with this branch E2E test fails in this pr. I tested it locally and functionality works as expected. Do you have any idea why E2E test fails after merge with main?  I have no idea why E2E fails.

If this pull request cant pass E2E test then I think we can close this pr, I will create another branch from main and submit new pull request. What you are suggesting?

**Review by parasharrajat — APPROVED** [review]
LGTM. don't worry about That test it's failing for some other reason.

**Pull request merged**
